db/migrate/20220830145530_create_events.rb

```diff
@@ -1,7 +1,7 @@
 class CreateEvents < ActiveRecord::Migration[5.2]
   def change
     create_table :events do |t|
-      t.datetime :date
```

Comment:
think we should leave this as datetime since datetime still returns a string. We can discuss this